spec/models/task_spec.rb

```diff
@@ -1,5 +1,43 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validation' do
+    it 'is valid with all attributes' do
+      user = User.create(email: "user@sample.com", password: "password")
```

Comment:
`@sample.com`は実在する可能性があるので、存在しないドメインである`@example.com`を使いましょう！
```suggestion
      user = User.create(email: "user@example.com", password: "password")
```

Reply:
810d06f